add-on/manifest.common.json

```diff
@@ -33,7 +33,10 @@
     "icons/png/ipfs-logo-off_38.png",
     "icons/png/ipfs-logo-off_128.png",
     "icons/ipfs-logo-on.svg",
-    "icons/ipfs-logo-off.svg"
+    "icons/ipfs-logo-off.svg",
```

Comment:
@whizzzkid  This file is a template used for generating final `add-on/manifest.json` for specific runtime: `firefox`, `chromium`, or `brave` (production brave used `chromium`, but we need this for local testing). 

Generation happens in `ci/update-manifest.sh` (triggered during build with some env vars). It was way more complex in the past when we had beta channels, we could simplify build pipeline these days, but did not bother since it works fine.